ApiBundle/EventSubscriber/AbstractSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\EventSubscriber;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\KernelEvent;
+
+/**
+ * Class AbstractSubscriber
+ */
+abstract class AbstractSubscriber
+{
+    protected $resolver;
+    protected $annotationReader;
+
+    /**
+     * @param AnnotationReader            $annotationReader
+     * @param ControllerResolverInterface $resolver
+     */
+    public function __construct(AnnotationReader $annotationReader, ControllerResolverInterface $resolver)
+    {
+        $this->resolver = $resolver;
+        $this->annotationReader = $annotationReader;
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return boolean
+     */
+    protected function isApiRequest(Request $request)
+    {
+        return 0 === strpos($request->get('_route'), 'open_orchestra_api');
```

Comment:
We get a problem here. Not only on this PR but generally with that test (it is also used in the serialiser subscriber) :
to extend the api in others bundles (generic bundles or app bundles), api routes must be prefixed by open_orchestra_api, which is not logical in that cases.
Can you find a more generic test ?